recipes/bfiocpp/bld.bat

```diff
@@ -0,0 +1,32 @@
+set CMAKE_GENERATOR_PLATFORM=
+set CMAKE_GENERATOR_TOOLSET=
+set CMAKE_GENERATOR=Ninja
+
+set CMAKE_USE_SYSTEM_LIBS=-DTENSORSTORE_USE_SYSTEM_AOM=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_AVIF=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_BENCHMARK=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_BLOSC=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_BROTLI=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_BZIP2=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_CARES=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_CURL=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_DAV1D=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_JPEG=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_LIBLZMA=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_LIBYUV=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_LZ4=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_NGHTTP2=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_NLOHMANN_JSON=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_OPENSSL=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_PNG=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_SNAPPY=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_TIFF=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_WEBP=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_ZLIB=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DTENSORSTORE_USE_SYSTEM_ZSTD=ON
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DAVIF_ENABLE_GTEST=OFF
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DBENCHMARK_ENABLE_GTEST_TESTS=OFF
+set CMAKE_USE_SYSTEM_LIBS=%CMAKE_USE_SYSTEM_LIBS% -DINSTALL_GTEST=OFF
+
+set CMAKE_ARGS=-DCMAKE_PREFIX_PATH=%PREFIX% -DCMAKE_INSTALL_PREFIX=%PREFIX% %CMAKE_USE_SYSTEM_LIBS%
```

Comment:
conda-forge already comes with a pre-set CMAKE_ARGS that contains the necessary values
```suggestion
set CMAKE_ARGS=%CMAKE_ARGS% %CMAKE_USE_SYSTEM_LIBS%
```